programs/vote/src/vote_state.rs

```diff
@@ -151,6 +157,14 @@ pub struct VoteState {
     pub votes: VecDeque<Lockout>,
     pub root_slot: Option<u64>,
 
+    /// the signer for vote transactions
+    authorized_voter: BTreeMap<u64, Pubkey>,
```

Comment:
This is sorted so doesn't break consistency but make sure it doesn't grow unbounded.

Reply:
It can only grow as large as the number of epochs that we're using for the `leader_schedule_offset` + 1, so by default should be at most 2